### PR TITLE
Adding morningstar-data package

### PR DESCRIPTION
**Pull request by mstar-ruifeng**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/morningstar-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "morningstar-data" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: morningstar-data 
+  version: 1.3.0 
+
+source:
+  url: https://files.pythonhosted.org/packages/cd/dd/b90be5d468e9808a69cdea7fee16cd99a5da5eb1e29f5479954b0de93754/morningstar_data-1.3.0.tar.gz
```
Comment (xhochy) [inline recipes/morningstar-data/meta.yaml:9]: Please use a different url where you can template the version. Pick the pattern from any other Python package.

File: recipes/morningstar-data/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 0
+
+requirements:
+  build:
+    - python >=3.8
+    - pip
```
Comment (xhochy) [inline recipes/morningstar-data/meta.yaml:20]: Not needed 
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/morningstar-data/meta.yaml (hunk 2 above)
Comment (mstar-ruifeng) [reply recipes/morningstar-data/meta.yaml]: Removed. 
Thank you!

**Comment by mstar-ruifeng** [comment]
Hi @xhochy , could you please review the changes?

File: recipes/morningstar-data/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.8.0,<4.0.0
+    - pandas >=1.5.3,<1.6.0
+    - typeguard >=2.13.3,<3.0.0
+    - pyjwt >=2.4.0,<3.0.0
+    - boto3 >=1.20.1,<2.0.0
+    - cryptography 40.0.2
```
Comment (xhochy) [inline recipes/morningstar-data/meta.yaml:28]: This seems overly tight. As you are part of upstream: please loosen that in future.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/morningstar-data/meta.yaml (hunk 3 above)
Comment (mstar-ruifeng) [reply recipes/morningstar-data/meta.yaml:28]: Updated. Thank you!

**Review by xhochy — APPROVED** [review]

**Pull request merged**